app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from datetime import date
+
+from app.errors import NotVaccinatedError
+from app.errors import NotWearingMaskError
+from app.errors import OutdatedVaccineError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date and expiration_date < date.today():
+            raise OutdatedVaccineError
+        if not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError

```

Comment:
The `NotWearingMaskError` should be raised with a descriptive message. Ensure that the exception includes a message as per the checklist requirements.